plugins/auth-backend/api-report.md

```diff
@@ -94,6 +97,20 @@ export class IdentityClient {
     }>;
     }
 
+// @public (undocumented)
+export const microsoftDefaultSignInResolver: SignInResolver<OAuthResult>;
```

Comment:
Let's not export this one, as its behaviour isn't really something that we want to use in the future and we just need it for backwards compatibility